bitwarden_license/src/Sso/Utilities/DynamicAuthenticationSchemeProvider.cs

```diff
@@ -412,7 +411,7 @@ private DynamicAuthenticationScheme GetSaml2AuthenticationScheme(string name, Ss
             }
             if (!string.IsNullOrWhiteSpace(config.IdpX509PublicCert))
             {
-                var cert = config.IdpX509PublicCert.Base64StringToByteArray();
+                var cert = Convert.FromBase64String(config.IdpX509PublicCert);
```

Comment:
As discussed the changes look good, just would like Kyle to verify the change of this behavior.

Reply:
Checked in with Kyle, and we already have helpers for this in CoreHelpers. Replaced the functionality with the correct helpers. The naming of the libs method confused me, but it seems to be url encoded base64. not regular base64.

Reply:
Gotcha, good thing we checked.